src/workflows-app/SubmissionConfig.js

```diff
@@ -183,9 +182,10 @@ export const BaseSubmissionConfig = (
 
           if (cbasProxyUrlState.status === AppProxyUrlStatus.Ready) {
             loadRunSet(cbasProxyUrlState.state).then((runSet) => {
-              setRunSetRecordType(runSet.record_type);
```

Comment:
Do we no longer set the `runSetRecordType`? I think we might still want to keep this in state, but when passing the record type on L187 use `runSet.record_type` as you have it now so that `undefined` is not passed

Reply:
This state variable was only being set on page load and the information was being duplicated by `selectedRecordType` (which is the value that is actually updated and sent along in the eventual request). Because it was only used for the initial load (and had the same value as the selectedRecordType) there doesn't seem to be any motivation to keep it in.

Reply:
Ah I see. Thank you for the clarification 👍